src/components/Contributors/404.js

```diff
@@ -0,0 +1,7 @@
+// contributors 404
+export const contributorsNotFound = [
+  'bartushek',
+  'rynclark',
+  'henriquea',
+  'makuzaverite',
```

Comment:
Hardcoding them is not a good idea. Is there any alternative?

Reply:
What's your suggestion other than maintaining a list manually? I can think of reading the list from github API, however there're thousands of contributors on this repository, it just doesn't make sense to read it every time we build the site because most of the time the list should stay the same.

Reply:
Hmm, reading API everytime doesn't makes sense. Perhaps another reason for removing this section in favor of a separate contributors page. https://github.com/webpack/webpack.js.org/issues/4441